Tests/KernelTests/main.mm

```diff
@@ -462,23 +436,27 @@ - (void)clearCacheTests {
   [cat launch];
   [cat waitUntilExit];
 
-  if (self.timesSeenCat != 2) {
+  if (timesSeenCat != 2) {
     TFAIL();
   }
 
   TPASS();
 }
 
-/// Tests that the kernel still denies blocked binaries even if launched while traced
 - (void)blocksDeniedTracedBinaries {
   TSTART("Denies blocked processes running while traced");
 
+  self.handlerBlock = ^santa_action_t(santa_message_t msg) {
+    if (strncmp("/bin/mv", msg.path, 7) == 0) return ACTION_RESPOND_DENY;
+    return ACTION_RESPOND_ALLOW;
+  };
+
   pid_t pid = fork();
   if (pid < 0) {
     TFAILINFO("Failed to fork");
   } else if (pid > 0) {
     int status;
-    waitpid(pid, &status, 0);
+    while (waitpid(pid, &status, 0) != pid) {} // handle EINTR
```

Comment:
Can end empty while with semi-colon instead of {}

Reply:
Good catch